### PR TITLE
Fix unit price styling on order detail page

### PR DESCRIPTION
**Pull request by tauthomas01**
**Why are these changes introduced?**

Fixes https://github.com/Shopify/dawn/issues/4

The goal of this PR is to fix the unit price styling on the order page:

> Font size on customer order details page is inconsistent with the font size used for unit pricing throughout the rest of the theme. [Screenshot](https://screenshot.click/21-06-wma1t-itru4.png) 

image

image

**What approach did you take?**

- Followed similar approach as other styling on order page: use native CSS pseudo element and target HTML elements directly. In this case, I'm targeting the 3rd column of the table body, and target the 2nd `dd` element (which is the unit price)

**Demo links**

- [Store](https://os2-demo.myshopify.com/?preview_theme_id=120839897110)
- [Editor](https://os2-demo.myshopify.com/admin/themes/120839897110/editor)

**Checklist**
- [x] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [x] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [x] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [x] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [x] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

**Review by sofiamatulis — DISMISSED** [review]
Nice 👍 Tested it and code looks good. 

Some other CSS properties were changed, but I think they are now in the correct format 🤔

**Review by chrisberthe — DISMISSED** [review]
Can you hold off until #97 is merged? You'll need to rebase once this is done, cc @KaichenWang

**Review by chrisberthe — APPROVED** [review]

**Review by sofiamatulis — APPROVED** [review]

**Pull request merged**
